build-logic/testing-convention/src/main/kotlin/convention.gradle-testing.gradle.kts

```diff
@@ -34,12 +34,40 @@ val gradleTestTask = tasks.register<Test>("gradleTest") {
 
     useJUnitPlatform()
 
-    @Suppress("MagicNumber")
-    maxParallelForks = 8
+    /**
+     * timings provided only as relative to each other
+     * gradleTest:
+     *  1: 2m14s
+     *  16: 3m16s
+     */
+    maxParallelForks = 1
+
+    /**
+     * usually there is a small amount of test classes per module, and tests are not so memory hungry
+     */
+    setForkEvery(null)
```

Comment:
> usually

How about protecting from it by setting a big enough value?
Will it affect execution when it's greater than a usual count of tests in a module?

Reply:
it was a default before;
i saw no impact setting it to 50

i don't want to change it in this PR